src/main/java/command/UserInput.java

```diff
@@ -0,0 +1,130 @@
+package command;
+import java.util.Scanner;
+
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.ToDo;
+
+import java.util.InputMismatchException;
+import java.util.List;
+
+public class UserInput {
+    public static void parseInput(String input, Scanner in, List<Task> taskList) {
+
+        int taskID;
```

Comment:
Variable names must be in camelCase. So maybe name this `taskId` instead. 

https://se-education.org/guides/conventions/java/basic.html#naming